EasyInventory/src/main/java/com/easyinventory/enterprise/controller/EasyInventoryController.java

```diff
@@ -18,45 +18,45 @@
 public class EasyInventoryController {
 
     @Autowired
-    EasyInventoryService easyInventoryService;
+    private EasyInventoryService easyInventoryService;
```

Comment:
Based upon java standards, I opted to add a private access modifier to this variables, as they are only accessed in this class. 

https://docs.oracle.com/javase/tutorial/java/javaOO/variables.html